src/lib/icons.tsx

```diff
@@ -41,6 +41,121 @@ export const IconSvgs = {
       />
     </svg>
   ),
+  fblogo: (
```

Comment:
should we update the name to `facebook` so that it's consistent with how you named the other logos?